OpenSim/Simulation/Test/testInitState.cpp

```diff
@@ -50,8 +50,10 @@ int main()
     try {
         LoadOpenSimLibrary("osimActuators");
         testStates("arm26.osim");
-        testMemoryUsage("arm26.osim");
-        testMemoryUsage("PushUpToesOnGroundWithMuscles.osim");
+        if (isGetRSSValid()) {
```

Comment:
I don't think this is a good idea, as it will just silently skip the tests if `!isGetRSSValid()`. My understanding is that part of the motivation behind his PR was to be more strict about testing memory usage (e.g., not just allowing the test to pass if `mem1 < mem0`). If you want to check whether `getCurrentRSS()` is operating as expected, I suggest moving this part of the test to a new test case and throwing an exception if `!isGetRSSValid()`. We could then decide to merge a PR even if `getCurrentRSS()` malfunctions, but at least we would have that information.